src/Components/PokemonList.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { useGetListOfPokemonQuery } from '../redux/pokemonApi'
+import Pokemon from './Pokemon'
+
+
+
+const PokemonList = () => {
+	const {data,error,isLoading} =useGetListOfPokemonQuery()
+	const [type, setType] = useState("all")
+
+	const changeType=(type)=>{
+		setType(type)
+	}
+
+	return (
+		<div>
+			{/* <div className='flex'>
+					<button onClick={()=>changeType('all')}>All</button>
+					<button onClick={()=>changeType('water')}>Water</button>
+					<button onClick={()=>changeType('fire')}>Fire</button>
+
+			</div> */}
+		
+		<div className='m-auto p-4 grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 place-items-center  bg-slate-100 grid gap-6 container border-2 w-screen'>
+			
+			{isLoading ? (<div>Data is Loading</div>)
+			: data ? (
+				
+				data.map((name)=>{
+					return <Pokemon key={name} name={name}></Pokemon>
+				})
+			)	: null
+		}
+
+
+		</div>
```

Comment:
- [ ] Please ensure to stick to the typography, layout, and spacing between cards as shown in the design guidelines below
### Design Guideline
![image](https://user-images.githubusercontent.com/46839548/212501250-2d457d89-112c-413c-89f2-5e39a37386a5.png)
### Your Implementation
![image](https://user-images.githubusercontent.com/46839548/212501308-61def3c3-74ad-45d8-948d-bc5c05a13f87.png)

Reply:
Fixed.